Makefile

```diff
@@ -16,6 +16,10 @@ demo:
 demo-clean:
 	docker-compose -f docker-compose.demo.yml rm -f -s
 
+.PHONY: test
```

Comment:
ooh i didnt know you could do it like this instead of one big blob of phonies at the top of the makefile, very nice

Reply:
Yeah. This is how I prefer to do it. To avoid orphaned phonies! 😆